QuantConnect.Polygon/PolygonDataProvider.cs

```diff
@@ -175,16 +179,13 @@ public void SetJob(LiveNodePacket job)
                 throw new ArgumentException("The Polygon.io API key is missing from the brokerage data.");
             }
 
-            _apiKey = apiKey;
-
-            var maxSubscriptionsPerWebSocket = 0;
             if (!job.BrokerageData.TryGetValue("polygon-max-subscriptions-per-websocket", out var maxSubscriptionsPerWebSocketStr) ||
-                !int.TryParse(maxSubscriptionsPerWebSocketStr, out maxSubscriptionsPerWebSocket))
+                !int.TryParse(maxSubscriptionsPerWebSocketStr, out var maxSubscriptionsPerWebSocket))
             {
                 maxSubscriptionsPerWebSocket = -1;
             }
```

Comment:
This I think is a small bug. The default value for `maxSubscriptionsPerWebSocket` is -1. Could you please adapt it so that is `job.BrokerageData` doesn't have it or it can't be parsed as an integer, the default value is -1 always and not 0?